app/views/restaurants/show.html.erb

```diff
@@ -0,0 +1,12 @@
+<p><%= link_to "Back to Restaurant list", restaurants_path %></p>
```

Comment:
🔨 @sbwheeler indentation is off, each level of indentation should only be 2 spaces